forui/test/src/widgets/button/button_spinner_golden_test.dart

```diff
@@ -0,0 +1,104 @@
+@Tags(['golden'])
+library;
+
+import 'package:flutter/widgets.dart';
+
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:forui/forui.dart';
+import 'package:forui/src/widgets/button/button.dart';
+import '../../test_scaffold.dart';
+
+void main() {
+  group('FButtonSpinner', () {
+    group('blue screen', () {
+      testWidgets('FButtonContent', (tester) async {
+        await tester.pumpWidget(
+          TestScaffold.blue(
+            child: Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 20),
+              child: FButton(
+                label: const Text('Button'),
+                style: TestScaffold.blueScreen.buttonStyles.primary,
+                prefix: const FButtonSpinner(),
+                suffix: FIcon(FAssets.icons.circleStop),
+                onPress: () {},
+              ),
+            ),
+          ),
+        );
+
+        await expectLater(find.byType(TestScaffold), isBlueScreen);
+      });
+
+      testWidgets('FButtonIconContent', (tester) async {
+        await tester.pumpWidget(
+          TestScaffold.blue(
+            child: Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 20),
+              child: FButton.icon(
+                style: TestScaffold.blueScreen.buttonStyles.primary,
+                child: const FButtonSpinner(),
+                onPress: () {},
+              ),
+            ),
+          ),
+        );
+
+        await expectLater(find.byType(TestScaffold), isBlueScreen);
+      });
+    });
```

Comment:
I don't think we need this since we're testing whether FButtonSpinner uses the correct color, not whether FButton.icon passes down the correct color.